README

```diff
@@ -0,0 +1,53 @@
+OMAudioLib
+
+OMAudioLib is a Juce-based audio playback library providing a C API
+for use in music applications.
+
+-------------------------------------
+
+Compilation
+
+To build OMAudioLib, get the sources from
+
+  https://github.com/openmusic-project/omaudiolib
+
+The 3 platforms Linux, MacOSX and Windows have files for build systems
+(Makefile, XCode, VisualStudio) inside the Builds directory.  So
+fex. to build your own version of OMAudioLib.so for Linux, you'd do
+something like
+
+  cd Builds/Linux
+  make
+
+-------------------------------------
+
+Notes for maintainers on Linux, dependencies, bugs between OS'es
+
+OMAudioLib is based on JUCE, and some of the core Juce-modules are
+dependent on libs with different default variants between OS'es
+(Fedora: libcurl, Ubuntu: libcurl-gnutls), while none of these libs'
+symbols are ever used by OM.
+
+To compile OMAudioLib.so so that it doesn't break because of these
+inconsistencies between OS'es, edit the JUCE_LDFLAGS lines down in the
+Makefile, adding:
+
+  -Wl,--as-needed $(shell pkg-config --libs alsa libcurl)
+
+-------------------------------------
+
+Using OMAudioLib in OM
+
+OM already comes with a precompiled version of OMAudioLib installed as
+part of OMs resources.  If you compile your own version and want to
+use that with OM, install it where OM already has installed its
+version of OMAudioLib, taking care to backup the original version.
+
+-------------------------------------
+
+Bug reports
+
+If you observe strange behavior, crashes, warnings etc, please check
```

Comment:
people reading this are trying to build the lib, so I'd say that it makes more sense to report issues on the issue tracker rather that the user forum... ?

Reply:
Ok, i've removed the link to the forum.

On Linux, the problems people have had are mainly incompatibilities with already installed libs, or confusion about where to place things.  Once they download omaudiolib and build themselves, everything seems ok.